db/migrate/20170718111834_unify_container_definition_and_container.rb

```diff
@@ -0,0 +1,174 @@
+class UnifyContainerDefinitionAndContainer < ActiveRecord::Migration[5.0]
+  class ContainerDefinition < ActiveRecord::Base
+    has_many :container_port_configs, :class_name => 'UnifyContainerDefinitionAndContainer::ContainerPortConfig'
+    has_many :container_env_vars, :class_name => 'UnifyContainerDefinitionAndContainer::ContainerEnvVar'
+    has_one :security_context, :as => :resource
```

Comment:
This doesn't look like it's being used, so can we remove it?  I'm asking because polymorphic types end up causing issues when accessed through has_many/belongs_to, but luckily you are using resource_type/resource_id directly in the migrations below.